xkcd_dl/cli.py

```diff
@@ -236,23 +228,22 @@ def set_custom_path(custom_path):
         print("The path does not exist.")
     return path_was_set
 
+
 def show_xkcd(num):
     download_one(read_dict(), num)
-    path = '{current_directory}/xkcd_archive/{name}/'.format(
-        current_directory=WORKING_DIRECTORY, 
-        name=num
-    )
+    path = '{current_directory}/xkcd_archive/{name}/'.format(current_directory=WORKING_DIRECTORY, name=num)
```

Comment:
Retracted the change and pushed the modified one. Let me know if you have any other concerns